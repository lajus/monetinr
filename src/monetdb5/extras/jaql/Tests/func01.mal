# test of groupable functions
jaql.x("[1,2,3] -> sum();");
jaql.x("[1,2,3] -> avg();");
jaql.x("sum([1.2,2.4,3.6]);");
jaql.x("avg([1.2,2.4,3.6]);");
jaql.x("count([1.2,5,3.6,8,\"bla\"]);");

# groupable, but with singleton arguments
jaql.x("range(6);");
jaql.x("range(4,8);");
jaql.x("range(3,11,3);");
