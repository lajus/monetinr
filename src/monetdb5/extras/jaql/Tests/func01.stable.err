stderr of test 'func01` in directory 'monetdb5/extras/jaql` itself:


# 20:16:10 >  
# 20:16:10 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "gdk_dbfarm=/net/sofia.ins.cwi.nl/export/scratch1/fabian/tmp/mtest-jacqueline-sofia.ins.cwi.nl/five/dbfarm" "--set" "mapi_open=true" "--set" "mapi_port=37633" "--set" "monet_prompt=" "--trace" "--forcemito" "--set" "mal_listing=2" "--dbname=mTests_extras_jaql" "func01.mal"
# 20:16:10 >  

# builtin opt 	gdk_dbname = demo
# builtin opt 	gdk_dbfarm = /ufs/fabian/scratch/ssd/monetdb/jacqueline/program-x86_64/var/lib/monetdb5/dbfarm
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_alloc_map = no
# builtin opt 	gdk_vmtrim = yes
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	gdk_dbfarm = /net/sofia.ins.cwi.nl/export/scratch1/fabian/tmp/mtest-jacqueline-sofia.ins.cwi.nl/five/dbfarm
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 37633
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	gdk_dbname = mTests_extras_jaql

# 20:16:10 >  
# 20:16:10 >  "Done."
# 20:16:10 >  

