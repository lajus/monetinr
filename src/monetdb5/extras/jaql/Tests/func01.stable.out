stdout of test 'func01` in directory 'monetdb5/extras/jaql` itself:


# 20:16:10 >  
# 20:16:10 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "gdk_dbfarm=/net/sofia.ins.cwi.nl/export/scratch1/fabian/tmp/mtest-jacqueline-sofia.ins.cwi.nl/five/dbfarm" "--set" "mapi_open=true" "--set" "mapi_port=37633" "--set" "monet_prompt=" "--trace" "--forcemito" "--set" "mal_listing=2" "--dbname=mTests_extras_jaql" "func01.mal"
# 20:16:10 >  

# MonetDB 5 server v11.8.0 "jacqueline-6964bb75810c"
# Serving database 'mTests_extras_jaql', using 8 threads
# Compiled for x86_64-pc-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 15.630 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2013 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://sofia.ins.cwi.nl:37633/
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded
function user.main():void;
# test of groupable functions 
    jaql.x("[1,2,3] -> sum();");
    jaql.x("[1,2,3] -> avg();");
    jaql.x("sum([1.2,2.4,3.6]);");
    jaql.x("avg([1.2,2.4,3.6]);");
    jaql.x("count([1.2,5,3.6,8,\"bla\"]);");
# groupable, but with singleton arguments 
    jaql.x("range(6);");
    jaql.x("range(4,8);");
    jaql.x("range(3,11,3);");
end main;
[ 6 ]
[ 2.000000 ]
[ 7.200000 ]
[ 2.400000 ]
[ 5 ]
[ 0, 1, 2, 3, 4, 5 ]
[ 4, 5, 6, 7, 8 ]
[ 3, 6, 9 ]


# 20:16:10 >  
# 20:16:10 >  "Done."
# 20:16:10 >  

